core/src/main/java/io/questdb/griffin/engine/table/AsyncFilteredNegativeLimitRecordCursor.java

```diff
@@ -176,6 +181,8 @@ private void fetchAllFrames() {
                 }
 
                 frameSequence.collect(cursor, false);
```

Comment:
Could you add test(s) to exercise this branch ?

Reply:
This LoC is covered with `LimitTest.testNegativeLimitMultiplePageFramesPartitioned`. Maybe you're referring to something else?

Reply:
I meant line 185 . I try to put comment above the code it refers to and looks fine in "Files changed" view but on main PR page it actually shows a few lines above it .

Reply:
So, it's the `No frames to filter` branch. Added a test to cover it in b8101e3e434024f2ce2f5a290c6ab238ab08ae62